dds/src/implementation/rtps/messages/submessage_elements.rs

```diff
@@ -32,7 +32,7 @@ pub struct GuidPrefixSubmessageElement {
 
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub struct EntityIdSubmessageElement {
-    pub value: [u8; 4],

```

Comment:
So what we say by this is that an EntityIdSubmessageElement must contain a valid EntityId rather than just any collection of 4 elements of u8, right? Seems somehow better than what was there before even though given that the same stuff is implemented on both types (as I mentioned on my other comment) makes me wonder how useful this type really is in that case.